ST_progressBar.c

```diff
@@ -0,0 +1,120 @@
+/**************************************************************************/
+/* ST_progressBar.c
+    Function definitions for a progress bar printed to the terminal.
+    See ST_progressBar.h for a description of how to add a new Status.
+ */
+/**************************************************************************/
+
+#include "ST_progressBar.h"
+#include "ST_defines.h"
+#include "ST_globals.h"
+#include<string.h>
+
+/*************** Local Function(s). Treat these as private. ***************/
+
+double _calculateProgress(int innerLoopIteration, int outerLoopIteration, Status status);
+double _calculateInitializationProgress(int year);
+double _calculateSimulationProgress(int year, int iteration);
+
+/*********************** Function Definitions *****************************/
+
+/* Log the program's progress using a progress bar.
+	Param iteration: integer greater than 0. Input 0 iff the program is not currently in an iteration loop.
```

Comment:
@kpalmqui this actually isn't a typo: "iff" is short for "if and only if" which is [used in math](https://en.wikipedia.org/wiki/If_and_only_if) to mean "both are true or both are false". I can understand how this would be confusing to people from other sciences so I'll use the full expression. Sorry for the confusion!

Reply:
@ chaukap ok thanks!